frontend/src/main.tsx

```
@@ -1,9 +1,11 @@
+import { StrictMode } from "react";
+import { createRoot } from "react-dom/client";
 
-const container = document.getElementById('app');
-if (!container) {
-  throw new Error('Root element "#app" not found');
-}
-render(<App />, container);
+import "@/index.css";
+import { App } from "@/app";
+
+createRoot(document.getElementById("root")!).render(
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)